src/main/java/seedu/address/model/tag/SkillLevel.java

```diff
@@ -0,0 +1,8 @@
+package seedu.address.model.tag;
+
+/**
+ * Represents the different skill level a person can have.
+ */
+public enum SkillLevel {
+    BASIC, INTERMEDIATE, ADVANCE
```

Comment:
perhaps we should change advance to advanced? i.e basic skill level, advanced skill level

Reply:
regarding this, should we put this inside the relevant tags class? i don't want to pollute the file directory. if needed, we can extract them later. 👍

```
public class MedicalQualificationTag {
  public static enum SkillLevel { ... } 
}
```

in your tests, you can change it to `MedicalQualificationTag.SkillLevel.Advanced` instead

Reply:
Will do so